README.md

```diff
@@ -75,11 +75,15 @@ Possible values for the CM variable are:
 * `puppet` - Install Puppet
 * `salt`  - Install Salt
 
-You can also specify a variable `CM_VERSION`, if supported by the
-configuration management tool, to override the default of `latest`.
+You can also specify a variable `CM_VERSION` for all configuration management
+tools except Salt, to override the default of `latest`.
```

Comment:
Please see the comment in the review for `script/cmtool.bat`. I think we might be able to still preserve the functionality of allowing the user to set the salt version via `CM_VERSION`.